support/pages/filesPage.ts

```diff
@@ -24,7 +24,10 @@ export class FilesPage {
 
     await Promise.all([
       this.page.waitForResponse(
-        (resp) => resp.status() === 201 && resp.request().method() === 'MKCOL'
+        (resp) =>
+          resp.url().includes('graph/v1.0/drives/') &&
+          resp.status() === 200 &&
+          resp.request().method() === 'GET'
```

Comment:
why do you change it? when we extract zip we create folder and we should sent MKCOL or are there cases where we don't?

Reply:
The folder gets created before the extraction takes place. That means the test could be flaky in cases where the folder gets created, but the extraction is still running. The `GET` request on the other hand takes place at the very end after the extraction has finished.